### PR TITLE
Add private new tab page

### PR DESCRIPTION
**Pull request by cezaraugusto**
closes brave/brave-browser#200

Note: private search engine toggle is only aesthetic at this moment.

screen shot 2018-05-15 at 3 09 45 am

### CODE REVIEW

File: browser/ui/webui/new_tab_html_source.cc [hunk 1]
```diff
@@ -44,12 +44,68 @@ void CustomizeNewTabHTMLSource(content::WebUIDataSource* source) {
...
+    { "7dd94f3f5749c7866546c7694980ad76.woff2", IDR_BRAVE_TEXT_FONT_MULI_400_NORMAL_LATIN_EXT },
+    { "f0e1ee92a25c8246a4a1fc28b3f77084.woff2", IDR_BRAVE_TEXT_FONT_MULI_400_NORMAL_LATIN },
+    { "3bef707814df0c192a19df4f31518b26.woff2", IDR_BRAVE_TEXT_FONT_MULI_600_NORMAL_LATIN_EXT },
+    { "d0c20b36cf1fedcf75230805055f517b.woff2", IDR_BRAVE_TEXT_FONT_MULI_600_NORMAL_LATIN },
+    { "c0cf0365a6780870d455120fcc83b425.woff2", IDR_BRAVE_TEXT_FONT_MULI_700_NORMAL_LATIN_EXT },
+    { "03c3ebc1b575991ec582751b6b98e562.woff2", IDR_BRAVE_TEXT_FONT_MULI_700_NORMAL_LATIN },
+    { "6d23ab5117038380fd25f70909340a87.woff2", IDR_BRAVE_TEXT_FONT_MULI_800_NORMAL_LATIN_EXT },
+    { "ed853eeb06da3de8325c2a4abddc30c8.woff2", IDR_BRAVE_TEXT_FONT_MULI_800_NORMAL_LATIN },
+    { "7d0f2d0b966f31e8bc75bcf6954b6b49.woff2", IDR_BRAVE_TEXT_FONT_MULI_900_NORMAL_LATIN_EXT },
+    { "18a1df59976e7a18fed257368aaafa9f.woff2", IDR_BRAVE_TEXT_FONT_MULI_900_NORMAL_LATIN },
```
Comment (bbondy) [inline browser/ui/webui/new_tab_html_source.cc]: Could we only include the fonts that are used?

File: package.json [hunk 2]
```diff
@@ -18,6 +18,10 @@
     {
       "name": "Brian R. Bondy",
       "email": "brian@brave.com"
+    },
+    {
+      "name": "Cezar Augusto",
+      "email": "cezar@brave.com"
```
Comment (bbondy) [inline package.json]: 🎉

File: components/brave_new_tab_ui/components/app.js [hunk 3]
```diff
@@ -99,9 +101,7 @@ class NewTabPage extends React.Component {
     }
 
     if (this.props.newTabData.isIncognito) {
-      // TODO
-      // return <NewPrivateTab newTabData={newTabData} />
-      return null
+      return <NewPrivateTab stats={newTabData.stats} />
```
Comment (bbondy) [inline components/brave_new_tab_ui/components/app.js]: Height needs to be adjusted here so it takes up the whole height for the background.
screen shot 2018-05-22 at 1 04 36 pm

File: components/brave_new_tab_ui/components/clock.js [hunk 4]
```diff
@@ -1,48 +0,0 @@
...
-    window.setInterval(this.maybeUpdateClock.bind(this), 2000)
-  }
-  render () {
-    return <div className='clock'>
-      <span className='time'>{this.formattedTime}</span><span className='timePeriod'>{this.formattedTimePeriod}</span>
-    </div>
-  }
-}
-
-module.exports = Clock
```
Comment (bbondy) [inline components/brave_new_tab_ui/components/clock.js]: Nice to see this removed from here! 👍

File: components/brave_new_tab_ui/components/newPrivateTab.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+      return null
+    }
+
+    return (
+      <div>
+        <Page style={{
+          fontFamily: '"Poppins", sans-serif',
+          color: 'rgba(255,255,255,0.8)',
+          padding: '80px 60px 40px',
+          background: 'linear-gradient(#4b3c6e, #000)'
```
Comment (bbondy) [inline components/brave_new_tab_ui/components/newPrivateTab.js]: Should we have some of this factored out from the inline component structure?

File: components/brave_new_tab_ui/components/newPrivateTab.js [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                  text={window.loadTimeData.getString('privateNewTabDisclaimer1')} />
+                <Paragraph
+                  italic
+                  size='15px'
+                  text={window.loadTimeData.getString('privateNewTabDisclaimer2')} />
+                <BoxedContent margin='40px 0 25px'>
+                  <SwitchButton
+                    id='togglePrivateSearchEngine'
+                    size='large'
+                    checked={false /* TODO: enable search engine change */}
```
Comment (bbondy) [inline components/brave_new_tab_ui/components/newPrivateTab.js]: Let's put a dummy handler for now that calls to the reducer.  Currently when you click the switch:
screen shot 2018-05-22 at 1 16 56 pm

File: components/brave_new_tab_ui/components/stats.js [hunk 7]
```diff
@@ -53,31 +55,33 @@ class Stats extends React.Component {
...
+          <DataItem
+            color='#0796fa'
+            description={window.loadTimeData.getString('httpsUpgraded')}
+            counter={httpsUpgradedCount.toLocaleString()} />
+          <DataItem
+            color='#999999'
+            counter={timeSaved.value}
+            text={window.loadTimeData.getString(timeSaved.id)}
+            description={window.loadTimeData.getString('estimatedTimeSaved')} />
+        </DataBlock>
```
Comment (bbondy) [inline components/brave_new_tab_ui/components/stats.js]: Ditto some of these inline values, wondering if we should have a separate parameters js file or have the styles in css.

**Comment by cezaraugusto** [comment]
ya, re https://github.com/brave/brave-core/pull/105#pullrequestreview-122269055 seems like after an incremental build those files are added momentarily. Not sure what happened but removed from git. Other feedback addressed too, thanks!

**Comment by bbondy** [comment]
Nice work!

**Pull request merged**